pkg/cmd/setup.go

```diff
@@ -63,106 +64,115 @@ func setup(cmd *cobra.Command, args []string) {
 		utils.LogDebugError(err.Unwrap())
 	}
 
-	ignoreRepoConfig :=
-		// ignore when repo config is blank
-		(repoConfig.Setup.Project == "" && repoConfig.Setup.Config == "") ||
-			// ignore when project and config are already specified
-			(localConfig.EnclaveProject.Source == models.FlagSource.String() && localConfig.EnclaveConfig.Source == models.FlagSource.String())
-
-	// default to true so repo config is used on --no-interactive
-	useRepoConfig := true
-	if !ignoreRepoConfig && canPromptUser {
-		useRepoConfig = utils.ConfirmationPrompt("Use settings from repo config file (doppler.yaml)?", true)
-	}
-
-	currentProject := localConfig.EnclaveProject.Value
-	selectedProject := ""
-
-	switch localConfig.EnclaveProject.Source {
-	case models.FlagSource.String():
-		selectedProject = localConfig.EnclaveProject.Value
-	case models.EnvironmentSource.String():
-		utils.Log(valueFromEnvironmentNotice("DOPPLER_PROJECT"))
-		selectedProject = localConfig.EnclaveProject.Value
-	default:
-		if useRepoConfig && repoConfig.Setup.Project != "" {
-			utils.Print("Auto-selecting project from repo config file")
-			selectedProject = repoConfig.Setup.Project
-			break
+	for _, repo := range repoConfig.Setup {
+		expandedPath, _ := filepath.Abs(repo.Path)
+		scopedConfig = configuration.Get(expandedPath)
+
+		ignoreRepoConfig :=
+			// ignore when repo config is blank
+			(repo.Project == "" && repo.Config == "") ||
+				// ignore when project and config are already specified
+				(localConfig.EnclaveProject.Source == models.FlagSource.String() && localConfig.EnclaveConfig.Source == models.FlagSource.String())
+
+		// default to true so repo config is used on --no-interactive
+		useRepoConfig := true
+		if !ignoreRepoConfig && canPromptUser {
+			if len(repoConfig.Setup) > 1 && repo.Path != "" {
+				useRepoConfig = utils.ConfirmationPrompt(fmt.Sprintf("Use settings from repo config file (doppler.yaml) for %s?", expandedPath), true)
```

Comment:
Wondering how we should handle the case where the user incorrectly specifies a doppler.yaml with multiple values but no path. Like this:

```yaml
setup:
  - project: cli
    config: dev
  - project: example
    config: stg
```

Should we error when `len(repoConfig.Setup) > 1 && repo.Path == ""`?

Reply:
Added an error for this!